sources/HUBViewModelDiff.m

```diff
@@ -37,6 +38,12 @@
     return [indexPaths copy];
 }
 
+@interface  HUBViewModelDiff ()
+
+@property (nonatomic, assign) BOOL headerComponentIdentifierHasChanged;
```

Comment:
Wouldn't it make more sense to make this a public, readonly property? Since the rest of the diff model is immutable, and it might be useful for the consumer of the diff to see whether or not the header has been modified?